smak/Server/driver.cpp

```diff
@@ -26,10 +26,11 @@ int cclient(const std::shared_ptr<smak::User> &user,int id, smak::netController
         std::cout << "Server Received[" << msg <<"]\n";
 
         // Only the client who sent the message will get this response
-        user.get()->sendString("I got a message from you");
+        user.get()->sendString("_SERVER_: recvd message");
```

Comment:
I don't have a problem with standardizing these messages. Putting "__SERVER__" seems redundant. Furthermore, I just put this message here as a sanity check because not every OP was sending something back when I was working.

We should probably talk about standardizing messages and when we should be sending messages from the server to the client(s).